Dockerfile.QA

```diff
@@ -0,0 +1,20 @@
+FROM node:10 As build
+WORKDIR /app
+COPY package.json yarn.lock ./
+
+# Make yarn process packages
+RUN yarn install
+
+# Get all files
+COPY . .
+
+# Build project
+RUN yarn build:qa
+
+# Host project in nginx
+FROM nginx
+WORKDIR /app
+COPY --from=build /app/build /usr/share/nginx/html
+
+ENTRYPOINT ["nginx", "-g", "daemon off;"]
+EXPOSE 80 443
```

Comment:
Do you configure the real ports in swarm configuration?

Reply:
Exactly, we where thinking of adding it to readme or some other file to give some visibility to it.